src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -434,4 +434,36 @@
     <value>{0} {1}
     The default value '{1}' is not a valid value for {0}.</value>
   </data>
+  <data name="AddRefPostActionFailed" xml:space="preserve">
+    <value>Failed to add reference {0} to project file {1}</value>
+  </data>
+  <data name="AddRefPostActionMultipleProjFiles" xml:space="preserve">
+    <value>Found multiple possible project files in the same directory. Could not determine which to add a reference to. Files:</value>
```

Comment:
Actually, all of the project files are already output when that string is output.